tasks/task1/Task1b_Zijun_Liu.py

```diff
@@ -0,0 +1,60 @@
+#!/bin/env python3
+
+import sys
+
+import numpy as np
+
+
+def get_all_capitalized_word():
```

Comment:
This is missing documentation.
I would also encourage splitting this function up into 3 separate functions with declarative function naming.  This will give you the ability to reuse code and make it clearer to people who might be working on or using your code in the future to know what each section does.